### ISSUE TITLE
OIP5 - bigger whiteboard

### ISSUE THREAD

**Issue by ligi** [ethdevberlin/OIPS#5]
was just contacted by @anettrolikova - the organizer of @ethereum-magicians Berlin (for the one in BlockChainWeek around @ethberlinzwei).
They want to buy a big whiteboard and leave it in the ETH office afterwards. I think a bigger whiteboard would be nice and support this OIP - but want to feel the room and especially like to know if @mvglasenapp is OK with it

**Comment by mvglasenapp** [comment]
Sure, this sounds good to me. Can you maybe find out the dimensions of the white board @ligi ?

**Comment by ligi** [comment]
I don't (all Information I got is dumped here) - but @anettrolikova might

**Comment by anettrolikova** [comment]
Hi @mvglasenapp would be cool if you could help me find one (with wheels) that would fit into your office so we can buy it and then leave it at your office :)

**Comment by ligi** [comment]
would be great if the wheels are detachable - I was rather thinking we can hang it in the big conference room. With wheels it takes up more space in the room than hanging on the wall.

**Comment by mvglasenapp** [comment]
yes, please propose us one like what you had in mind, with detachable wheels, and we'll let you know whether it fits in the office @anettrolikova

**Comment by anettrolikova** [comment]
any ideas where to get a whiteboard from in Berlin ? :) I'm not located in Berlin

**Comment by ligi** [comment]
https://www.ebay-kleinanzeigen.de/s-friedrichshain/whiteboard/k0l3350

**Comment by anettrolikova** [comment]
thank you Ligi :) 
also @mvglasenapp what's the best way to chat with you ? are you on telegram or keybase ? riot works too
I have some questions for you.

**Comment by mvglasenapp** [comment]
@anettrolikova I've informed you on twitter :)

**Comment by franzihei** [comment]
Issue can be closed.
